src/js/video.js

```diff
@@ -736,19 +736,9 @@ videojs.insertContent = Dom.insertContent;
  */
 videojs.computedStyle = computedStyle;
 
-/*
- * Custom Universal Module Definition (UMD)
- *
- * Video.js will never be a non-browser lib so we can simplify UMD a bunch and
- * still support requirejs and browserify. This also needs to be closure
- * compiler compatible, so string keys are used.
- */
-if (typeof define === 'function' && define.amd) {
-  define('videojs', [], () => videojs);
+module.exports = videojs;
 
-// checking that module is an object too because of umdjs/umd#35
-} else if (typeof exports === 'object' && typeof module === 'object') {
-  module.exports = videojs;
-}
+videojs.use = middlewareUse;
+videojs.use('video/mp4', 'videojs/html5');
```

Comment:
Definitely something we need to think about. Because as it stands you can never add a middleware that handles `setSource` for 'video/mp4' since there is a tech registered as the first item always.